src/index.html

```diff
@@ -6,22 +6,442 @@
+              <p class="card__price-text">Price:</p>
+              <p class="card__price-count">$2,199</p>
+            </div>
+
+            <div class="card__button">
+              <a
+                href="index.html"
+                class="card__button-link"
+              >
+                Buy
+              </a>
+            </div>
+          </div>
+        </div>
+
+        <div class="card">
+          <img
+            src="images/imac.jpeg"
+            alt="'Imac computer"
+            class="card__img"
+          />
+
+          <div class="card__content">
+            <h2 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h2>
+            <p class="card__code">Product code: 195434</p>
+
+            <div class="card__review">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <p class="card__review-text">Reviews: 5</p>
+            </div>
+
+            <div class="card__price">
+              <p class="card__price-text">Price:</p>
+              <p class="card__price-count">$2,199</p>
+            </div>
+
+            <div class="card__button">
+              <a
+                href="index.html"
+                class="card__button-link"
+              >
+                Buy
+              </a>
+            </div>
+          </div>
+        </div>
+
+        <div class="card">
+          <img
+            src="images/imac.jpeg"
+            alt="'Imac computer"
+            class="card__img"
+          />
+
+          <div class="card__content">
+            <h2 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h2>
+            <p class="card__code">Product code: 195434</p>
+
+            <div class="card__review">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <p class="card__review-text">Reviews: 5</p>
+            </div>
+
+            <div class="card__price">
+              <p class="card__price-text">Price:</p>
+              <p class="card__price-count">$2,199</p>
+            </div>
+
+            <div class="card__button">
+              <a
+                href="index.html"
+                class="card__button-link"
+              >
+                Buy
+              </a>
+            </div>
+          </div>
+        </div>
+
+        <div class="card">
+          <img
+            src="images/imac.jpeg"
+            alt="'Imac computer"
```

Comment:
There is a typo in the alt attribute value. The single quote before 'Imac' should be removed to ensure proper accessibility and avoid confusion.